AppSupport/CPDistributedMessagingCenter.h

```diff
@@ -1,3 +1,8 @@
+#import <Foundation/NSObject.h>
+#import <Foundation/NSString.h>
+#import <Foundation/NSDictionary.h>
+#import <Foundation/NSError.h>
```

Comment:
Should we just `#import <Foundation/Foundation.h>` instead of each individual header? I think this is preferable in general as well as for module usage.

Reply:
It would simplify things, but it feels a bit like importing UIKit in that it's often importing much more than is needed. Don't really have a preference either way, but that was the motivation for these changes. Am fine to change it if desired